vnext/Microsoft.ReactNative.Cxx/JSValueReader.h

```diff
@@ -186,10 +165,10 @@ inline void ReadValue(IJSValueReader const &reader, /*out*/ std::wstring &value)
       value = reader.GetBoolean() ? L"true" : L"false";
       break;
     case JSValueType::Int64:
-      value = basic_string_convert<wchar_t>(reader.GetInt64());
+      value = std::to_wstring(reader.GetInt64());
       break;
     case JSValueType::Double:
-      value = basic_string_convert<wchar_t>(reader.GetDouble());
+      value = std::to_wstring(reader.GetDouble());
```

Comment:
Please replace it with the following code to make the unit test to pass:
``` C++
{
      std::wstringstream ss;
      ss << reader.GetDouble();
      value = ss.str();
      break;
}
```

Reply:
I have tried, but at least for me this leads to another error:
```
JSValueReader.h:182:5: error: cannot jump from switch statement to this case label
    default:
    ^
```
I am trying another approach ...